xml/System.Windows.Forms/ToolStrip.xml

```diff
@@ -2626,6 +2626,8 @@
           <format type="text/markdown"><![CDATA[  
```

Comment:
should we also add the default value to the return section for IntelliSense purposes and remove the possible values? If they add new enum values, this update would be easily missed.